### DISCUSSION TITLE
Error in testing hourglass: An example from Abaqus benchmark

### DISCUSSION THREAD

**Discussion by ZhengZQ007** [OpenRadioss/OpenRadioss#721] [Users]
To compare the technique to deal with the hourglassing of first-order uniform-strain elements between Abaqus explicit and radioss, I used an example from Abaqus benchmark. OpenRadioss failed to finish the calculation due to “**ERROR : TIME STEP LESS OR EQUAL DTMIN FOR SOLID ELEMENTS” when I select reduced-integration element formulations (Isolid=1, 2 and 24). But the full integration element formulations (Isolid=14, 17 and 18) can complete the calculation successfully. The attached files contain relevant Rad files. 
![image](https://user-images.githubusercontent.com/74033106/200153959-3c75e731-010c-4c2c-b507-a3c0c4ffa4a7.png)
[expl_bending.zip](https://github.com/OpenRadioss/OpenRadioss/files/9945152/expl_bending.zip)

**Comment by PaulAltair** [comment]
Adding a slightly more conservative timestep factor (0.8 rather than 0.9), it runs ok (using the Isolid = 24 as sent)
[DT-BRICK-0.8.zip](https://github.com/OpenRadioss/OpenRadioss/files/9946114/DT-BRICK-0.8.zip)

**Reply by ZhengZQ007** [reply]
Thanks very much.
